### ISSUE TITLE
Error restoring in Visual Studio: error NU1202: Package System.Text.Encoding.CodePages 4.5.0 is not compatible with net45 (.NETFramework,Version=v4.5).

### ISSUE THREAD

**Issue by ScottHutchinson** [fsprojects/ExcelProvider#78]
ExcelProvider.DesignTime.fsproj builds in Visual Studio 2019 version 16.8.3, but there are errors about incompatible nuget packages.

Also, when VS restores, it modifies the .paket\Paket.Restore.targets file, which is part of the repo and is complicating the source control. Maybe that file should be in .gitignore?

EDIT: Before building in VS, I ran build.cmd, which was successful.

What is the best way to resolve this version mismatch error?

There is a lot of paket and fake code that I don't understand well. We need someone who understands that to update this repo, so we can build without all of this noise from paket/nuget.

**Comment by quintusm** [comment]
Hi @ScottHutchinson 

I am in process of doing update to paket and fake code.  Unfortunately this is a fairly large undertaking as fake has moved from v4 to v5 that requires considerable porting as it has a lot of breaking changes to the api.  Also, I am having to figure out what the old code was doing, I was not around when it was done.  I cannot provide an estimate to completion time.

Please note that my intention also is to provide only a .netstandard 2 version in the update, as that seems to be the current best practice for type providers, giving maximum compatibility for the buck.

You might want to try using the netstandard2 version in the current live release?

**Comment by ScottHutchinson** [comment]
Thanks for the update. I was planning to add a new feature, but these errors distracted me. Maybe consider removing paket and fake, if they are more trouble than they are worth for this repo.

**Comment by quintusm** [comment]
I have updated the build and the tooling to new versions.  Also, the build is now only building for NetStandard2.0, which is getting rid of a lot of issues.

All packages have been updated to compatible versions.

@ScottHutchinson I presume you have moved on to other things, but if you are still looking at adding a feature it would be appreciated. You are welcome to discuss with me as I may be able to assist or it may overlap with feature already planned.

Am closing this issue for now as I believe it is resolved.
